bears/documentation/DocumentationStyleBear.py

```diff
@@ -134,6 +134,11 @@ def run(self, filename, file, language: str,
                 new_metadata, doc_comment.docstyle_definition,
                 doc_comment.marker, doc_comment.indent, doc_comment.position)
 
+            if file[doc_comment.range.start.line - 2] != '\n':
```

Comment:
this bear is defined as
```
LANGUAGES = {language for docstyle, language in
DocstyleDefinition.get_available_definitions()}
```

you put that there.  It is part of your project to support multiple languages (Java and Python according to https://gitlab.com/coala/GSoC-2017/issues/30).  The bear should be language agnostic.